buildshiprun/handler_test.go

```diff
@@ -123,19 +123,19 @@ func Test_GetImageName(t *testing.T) {
 		Output        string
 	}{
 		{
-			"Testcase1",
+			"Repository URL with default port for swarm",
 			"127.0.0.1:5000",
 			"registry:5000/username/function-name/",
 			"127.0.0.1:5000/username/function-name/",
 		},
 		{
-			"Testcase2",
+			"Repository URL with default port for kubernetes",
 			"127.0.0.1:31115",
 			"registry:31115/username/function-name/",
 			"127.0.0.1:31115/username/function-name/",
 		},
 		{
-			"Testcase3",
+			"Repository URL without port",
```

Comment:
This is better than before, but the input URLs are still quite confusing - as I'm not sure what is being tested here. What is GetImageName producing? The name for a push? Maybe we need to work on the function name too

Reply:
It is generating image name for the pull from the push image name and  `repository_url` env variable. Should we change it to `GetPullImageName`?